cli/src/ldap.rs

```diff
@@ -228,18 +228,26 @@ async fn main() {
                         }
                         println!("");
                     }
-                    for entry_uuid in &delete_uuids {
-                        println!("delete entryuuid: {}", entry_uuid);
-                    }
-                    if !present_uuids.is_empty() {
-                        println!("");
+                    if let Some(d_uuids) = &delete_uuids {
```

Comment:
`for entry_uuid in delete_uuids.as_ref().unwrap_or_default()` ?

Reply:
That's a neater solution for certain :)

Reply:
Also sorry for the yolo merge, I needed to put this in for a PR for kani that's quite important RN.

Reply:
Ah, no problem, do what you must, I'm just drive-by commenting :)

Reply:
They're good comments.